jetson/percep/camera.cpp

```diff
@@ -290,11 +306,13 @@ bool Camera::Impl::grab() {
 #if AR_DETECTION
 cv::Mat Camera::Impl::image() {
   std::string full_path = rgb_path + std::string("/") + (img_names[idx_curr_img]);
-cerr << img_names[idx_curr_img] << "\n";
-cerr << full_path << "\n";
+#if PERCEPTION_DEBUG
+  cout << img_names[idx_curr_img] << "\n";
```

Comment:
Can you make sure there are 4 spaces in the tab?

Reply:
Fixed